downstream/modules/platform/ref-system-requirements.adoc

```diff
@@ -100,13 +100,23 @@ h| Disk: service node | 40 GB dedicated hard disk space |
 * Storage volume should be rated for a minimum baseline of 1500 IOPS.
 * Projects are stored on control and hybrid, and for the duration of jobs, also on execution nodes. If the cluster has many large projects, consider having twice the GB in /var/lib/awx/projects, to avoid disk space errors.
 
-h| Disk: database node| 20 GB dedicated hard disk space |
+|Database node| Required | Notes
```

Comment:
This table isn't building out correctly for me--the column entries in the first row are all bolded, but the column headings aren't. Also, it looks like it's supposed to be multiple tables, but everything is stuck together. 
![Screen Shot 2023-07-17 at 16 30 06 PM](https://github.com/RedHatInsights/red-hat-ansible-automation-platform-documentation/assets/121137790/cdc0a3f5-5545-47a3-a459-0bc32ac29daf)